lib/ramble/ramble/software_environments.py

```diff
@@ -7,6 +7,10 @@
 # except according to those terms.
 
 from enum import Enum
+try:
+    import deprecation
+except ImportError:
```

Comment:
Do you think we can figure out which of these deprecation imports will happen first, and only warn on one of them? 

Ideally we would stop ramble hiding the regular import fail error. I added this print as a bandaid since we only did the important in one place, but the more places the uglier it gets (but perhaps we can just have a follow up task to sort that)

Reply:
When I uninstall 'deprecation' now, it seems like any command I try causes Ramble to error out on this one (SoftwareEnvironments.__init__()) rather than the others, so I'll leave this in and remove the warning from the rest

Reply:
Removed the try blocks per our chat discussion